### ISSUE TITLE
EinthusanTV

### ISSUE THREAD

**Issue by asaddon** [danamag/stremio-addons-list#744]
### Addon Manifest URL

https://einthusantv-k9mh.onrender.com/manifest.json

### Addon Description

Addon for Streaming movies from Einthusan.TV by Asaddon

### Language of Content

None

### Choose Labels

- [ ] anime
- [ ] asian drama
- [X] bollywood
- [ ] cartoons
- [ ] debrid support
- [X] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [X] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by asaddon** [comment]
@danamag Hello, my addon supports multiple languages, not just Hindi. It had a bug in manifest.json that I have fixed now. Could you please change the addon name to remove the language part?
Also, I’ve updated the version number. Will the details automatically refresh with this change aftewr some time? This is my first time pushing an addon here.

**Comment by asaddon** [comment]
@sleeyax @danamag Hello guys, any reason why my addon was removed as I'm not seeing it shown anywhere in the catalog.

**Comment by RSAK2105** [comment]
Hi, is it possible to develop new add-on for this site www.binged.com to fetch the OTT content search as similar to EinthusanTV?

**Comment by asaddon** [comment]
> Hi, its Not working for me, could you check ?

Hello, as per the logs everything is working as expected, what issue you're facing, is the addon catalog not loading or not streaming?

**Comment by glardz98** [comment]
I don't See catalog

**Comment by asaddon** [comment]
> I don't See catalog

Which language? I personally use Hindi and Punjabi and they're working perfectly, Also which device?

**Comment by glardz98** [comment]
How can i see on stremio, i use stremio app

**Comment by asaddon** [comment]
> How can i see on stremio, i use stremio app

On the app go to Addons page and check the installed Addons, there you can see which language you have it installed.

**Comment by asaddon** [comment]
> App is installed but i don't See movie

Go to the Discover page and check if you see Einthusan Addon there. 

![image](https://github.com/user-attachments/assets/b4e6d514-0a66-440d-ac50-699c98669257)

![image](https://github.com/user-attachments/assets/1aa4125d-11d1-4856-a917-7327430b8ac5)

**Comment by glardz98** [comment]
I don't See einthusan addon, but its installed

**Comment by glardz98** [comment]
![Image](https://github.com/user-attachments/assets/4ff2d270-b4a2-4116-99c2-24cf81bc954b)

**Comment by asaddon** [comment]
> I don't See einthusan addon, but its installed

Try removing and reinstalling it. 
Also check if it's showing on the Stremio Web App (Make sure you're logged in)

https://web.stremio.com/

**Comment by glardz98** [comment]
Same since https://web.stremio.com/ its Not working

**Comment by glardz98** [comment]
![Image](https://github.com/user-attachments/assets/9dc64ed9-8c09-47b1-9e6e-05edf42b343d)

**Comment by asaddon** [comment]
> ![Image](https://github.com/user-attachments/assets/9dc64ed9-8c09-47b1-9e6e-05edf42b343d)

I dont think you actually have the addon properly installed, I dont see any "Uninstall" Button here.
Go to the Addon website and install it properly.

https://einthusantv-k9mh.onrender.com/configure/

**Comment by glardz98** [comment]
Its working thanks, is or possible to put premium account ?

**Comment by asaddon** [comment]
> Its working thanks, is or possible to put premium account ?

Not needed for now, it's already using my premium account, I'll add that functionality if it's needed in future.

**Comment by rubanregmii** [comment]
i keep on installing addon with desired language but no change in app, what to do?

**Comment by asaddon** [comment]
> i keep on installing addon with desired language but no change in app, what to do?
> 

Doing it via mobile? Or Stremio Web?

**Comment by rubanregmii** [comment]
> > i keep on installing addon with desired language but no change in app, what to do?
> 
> Doing it via mobile? Or Stremio Web?

tried it on both, not working for me. I even unstalled and reinstalled the app .

**Comment by asaddon** [comment]
> > > i keep on installing addon with desired language but no change in app, what to do?
> > 
> > Doing it via mobile? Or Stremio Web?
> 
> tried it on both, not working for me. I even unstalled and reinstalled the app .
> 

Are you trying to install it directly from Stremio Addons page in the app? If yes then I think I know what's the issue is, I'll push a fix tomorrow night. 

In the meantime go directly to "https://einthusan.asaddon.com" select your desired language and install the addon.

**Comment by rubanregmii** [comment]
did as you said, now working 
thanks daa @asaddon

**Comment by ananthasaia6a6** [comment]
Hi @asaddon 
Einthusan streams are not working for me lately in mobile stremio app. Getting below error for every video. Tried to reinstall the addon still couldn't get rid of it. Please help

![Image](https://github.com/user-attachments/assets/c957fb34-642f-43bf-a77b-37256fcd94ac)

**Comment by asaddon** [comment]
> Hi @asaddon 
> Einthusan streams are not working for me lately in mobile stremio app. Getting below error for every video. Tried to reinstall the addon still couldn't get rid of it. Please help
> 
> ![Image](https://github.com/user-attachments/assets/c957fb34-642f-43bf-a77b-37256fcd94ac)

Hello, This error happens when Stremio is not able to connect to the video file server ( einthusan ) open https://einthusan.tv website directly in your mobile and see if you're able to play any videos. 

You can also try with a VPN.
